src/TLuaInterpreter.cpp

```diff
@@ -8874,6 +8874,26 @@ int TLuaInterpreter::createMapLabel(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#changeMapperSettings
+int TLuaInterpreter::changeMapperSettings(lua_State* L)
```

Comment:
Might I suggest deferring this Lua API addition to a separate PR that you can do later one?